spec/factories/entry.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
```

Comment:
i'm a fan of keeping all factories in a single file and keeping the file ordered alphabetically. especially for smaller data models such as this project's it shouldn't be too bad to navigate. i'd love to hear other's thoughts